### ISSUE TITLE
Twitch HTML5 player

### ISSUE THREAD

**Issue by Oxmaster** [igrigorik/videospeed#128]
Twitch is releasing HTML5 player so this plugin now works on the player so this brings up issues:

- with "Remember Playback Speed" the stream will start to buffer infinitely after it runs out when used with < 1 speed
- while writing in chat, backspace rewinds the stream even thought I don't have backspace binded to any function in chrome.

Solution:

- add blacklisting/whitelisting AND blacklist twitch by default

Video Speed Controller
Ver 0.3.2

Google Chrome	54.0.2803.0 (Official Build) canary (64-bit)
Revision	74ed884d4d42cb7152dc75fae33351e8e34220cd-refs/heads/master@{#406716}

**Comment by igrigorik** [comment]
> with "Remember Playback Speed" the stream will start to buffer infinitely after it runs out when used with < 1 speed

This is a bug in Twitch player.

> while writing in chat, backspace rewinds the stream even thought I don't have backspace binded to any function in chrome.

We don't bind anything to backspace, so this is either Twitch-native functionality or some other extension you have in Chrome.

> add blacklisting/whitelisting AND blacklist twitch by default

I'm not sure blacklist-by-default is necessary. We do have a separate issue for adding a user-defined blacklist (see https://github.com/igrigorik/videospeed/issues/121); once that lands you can disable twitch if you desire. 

Closing, let's continue the blacklist conversation in #121.

**Comment by Oxmaster** [comment]
How is the proper working of the plugin twitch's fault? If they are using HTML 5 player (to use it you have to have twitch turbo), then your plugin should work.

Now with the backspace bug, it was plugins fault and my fault. This was my config: http://i.imgur.com/hFB4FpB.png

It seems that the rewind button was somehow binded to backspace, after I binded it to any other key, the bug disappeared.

**Comment by igrigorik** [comment]
@Oxmaster infinite buffering on <1x playback is a bug in their implementation.
